template/addons/prisma/env-prisma-auth.js

```diff
@@ -0,0 +1,12 @@
+const { z } = require("zod");
+
+const envSchema = z.object({
+  DATABASE_URL: z.string().url(),
+  NODE_ENV: z.enum(["development", "test", "production"]),
+  NEXTAUTH_SECRET: z.string(),
+  NEXTAUTH_URL: z.string().url(),
+  GITHUB_ID: z.string(),
+  GITHUB_SECRET: z.string(),
+});
+
+module.exports.envSchema = envSchema;
```

Comment:
these can not be ESM, right?
```js
export const envSchema = ...
```

Reply:
Yes, but I think the `.mjs` might be supported